Shopist/Makefile

```diff
@@ -0,0 +1,15 @@
+all: secret
+
+LOCAL_XCCONFIG="./Shopist/shopist-secrets.local.xcconfig" 
+ifeq ($(secret),)
+	secret="my-secret-token"
+endif
+
+secret:
```

Comment:
- [ ] Also, we need to remove this from the root `Makefile`:
```
examples:
		@echo "⚙️  Generating 'examples/examples-secret.xcconfig' file..."
		./tools/config/generate-examples-config-template.sh
		@echo "OK 👌"
```
Which also means those two can be removed:
- [ ] `tools/config/examples-secret-template.xcconfig`
- [ ] `generate-examples-config-template.sh`